packages/lodestar/src/chain/stateTransition/util/attestation.ts

```diff
@@ -117,10 +116,8 @@ export function getAttestingIndices(
   bits: BitList
 ): ValidatorIndex[] {
   const committee = getCrosslinkCommittee(config, state, data.target.epoch, data.crosslink.shard);
-  assert(bits.bitLength === committee.length);
-
   // Find the participating attesters in the committee
-  return committee.filter((_, i) => bits.getBit(i)).sort((a, b) => a - b);
+  return committee.filter((_, i) => {return bits.getBit(i);}).sort((a, b) => a - b);
```

Comment:
why do we need the return here?

Reply:
we don't, I was logging something there and didn't return to original